packages/web/src/views/governance/components/governance-summary/GovernanceSummary.tsx

```diff
@@ -1,20 +1,72 @@
-import { GovernanceDetailData } from "./governance-detail/governance-detail-info/GovernanceDetailInfo";
-import GovernanceDetail from "./governance-detail/GovernanceDetail";
+import Link from "next/link";
+import React from "react";
+import { useTranslation } from "react-i18next";
 
-import { GovernanceSummaryWrapper } from "./GovernanceSummary.styles";
+import IconStrokeArrowRight from "@components/common/icons/IconStrokeArrowRight";
+import MissingLogo from "@components/common/missing-logo/MissingLogo";
+import { EXT_URL } from "@constants/external-url.contant";
+import { GovernanceSummaryInfo } from "@repositories/governance";
 
-interface GovernanceSummaryProps {
-  governanceDetailInfo?: GovernanceDetailData;
-  loading?: boolean;
+import GovernanceDetailInfo from "./governance-detail-info/GovernanceDetailInfo";
+
+import { GovernanceSummaryWrapper, TokenChip } from "./GovernanceSummary.styles";
+import { GNS_TOKEN } from "@common/values/token-constant";
+
+interface GovernanceDetailProps {
+  governanceSummary: GovernanceSummaryInfo;
+  isLoading: boolean;
 }
 
-const GovernanceSummary: React.FC<GovernanceSummaryProps> = ({
-  governanceDetailInfo,
-  loading,
-}) => (
-  <GovernanceSummaryWrapper>
-    <GovernanceDetail loading={loading} governanceDetailInfo={governanceDetailInfo} />
-  </GovernanceSummaryWrapper>
-);
+const GovernanceSummary: React.FC<GovernanceDetailProps> = ({
+  governanceSummary,
+  isLoading,
+}) => {
+  const {t} = useTranslation(); 
+  
+  return (
+    <GovernanceSummaryWrapper>
+      <div className="info-wrapper">
+        <GovernanceDetailInfo
+          title={t("Governance:summary.totalDel.title")}
+          value={
+            <>
+              {governanceSummary.totalDeligated.toLocaleString("en")}
+              <TokenChip>
+                <MissingLogo symbol="GNS" url={GNS_TOKEN.logoURI} width={24} />
```

Comment:
Why not use the symbol in the GNS_TOKEN variable?

Reply:
fix in https://github.com/gnoswap-labs/gnoswap-interface/pull/515/commits/ec46901323f50e3313bd1ee9b2c03984a64f0e39